packages/bento-design-system/src/util/defaultConfigs.tsx

```diff
@@ -483,7 +483,10 @@ export const underlineTabs: TabsConfig = {
     medium: 16,
     large: 32,
   },
-  lineHeight: 2,
+  lineHeight: {
+    medium: 2,
+    large: 4,
+  },
```

Comment:
is this a leftover, or is it deliberate?

Reply:
deliberate: we want to be able to define different widths based on the size of the component.

Reply:
It's ok to allow specifying a different height, but do we really want to change the value too? The more conservative option would be 
```suggestion
  lineHeight: {
    medium: 2,
    large: 2,
  },
```